### PR TITLE
Use the new electron.Notification API

### PR DESCRIPTION
**Pull request by sindresorhus**
https://github.com/electron/electron/blob/a6ea316a5de1964761a82d9376c23e99eff16d25/docs/api/notification.md

### CODE REVIEW

File: app/src/main/share-service-context.js [hunk 1]
```diff
@@ -29,14 +29,10 @@ export default class ShareServiceContext {
   }
 
   notify(text) {
-    // TODO: Add Kap icon as `icon` option
-    // TODO: Waiting for: https://github.com/electron/electron/pull/9269
-    // (new electron.Notification({title, body})).show();
-
-    electron.app.kap.mainWindow.send('show-notification', {
+    (new electron.Notification({
```
Comment (albinekb) [inline app/src/main/share-service-context.js]: Time to add the `icon`? https://github.com/electron/electron/blob/master/docs/api/notification.md#new-notificationoptions-experimental

File: package.json [hunk 2]
```diff
@@ -19,9 +19,9 @@
     "chalk": "1.1.3",
     "cssnano": "3.10.0",
     "del": "2.2.2",
-    "electron": "1.7.2",
-    "execa": "0.6.3",
+    "electron": "1.7.3",
```
Comment (albinekb) [inline package.json]: Do you need to upgrade [`electron-builder`](https://github.com/electron-userland/electron-builder/releases) at the same time? 🤔

Comment (sindresorhus) [inline package.json]: No, that's unrelated.

File: app/src/main/share-service-context.js (hunk 1 above)
Comment (sindresorhus) [inline app/src/main/share-service-context.js]: I considered it, but the app icon is used by default, so wasn't needed.

Comment (albinekb) [inline app/src/main/share-service-context.js]: aha nice! 👌

**Comment by albinekb** [comment]
Maybe use the new API here too? https://github.com/wulkano/kap/blob/master/app/src/main/share-service.js#L82

**Comment by sindresorhus** [comment]
> Maybe use the new API here too? https://github.com/wulkano/kap/blob/master/app/src/main/share-service.js#L82

Done

**Pull request merged**